src-ghc/Pact/Types/API.hs

```diff
@@ -98,7 +98,7 @@ instance FromJSON ApiResult where parseJSON = lensyParseJSON 3
 instance ToJSON ApiResult where toJSON = lensyToJSON 3
 
 -- | What you get back from a Poll
-newtype PollResponses = PollResponses (HM.HashMap RequestKey ApiResult)
+newtype PollResponses = PollResponses (HM.HashMap RequestKey ApiResult) deriving (Eq, Show)
```

Comment:
Put this on a newline please. In general this PR has very long lines

Reply:
Updated.